### ISSUE TITLE
Mutable/user-defined config location?

### ISSUE THREAD

**Issue by stevefan1999-personal** [siderolabs/extensions#235]
I don't know whether [https://github.com/siderolabs/extensions/issues/24#issuecomment-1735682704](https://github.com/siderolabs/extensions/issues/24#issuecomment-1735682704) would be contradicting with the philosophy of Talos because everything is supposed to be running off the SquashFS, but some users may want to edit the config file. Including those config files are quite long to fit into YAML.

I'm not aware if the user can supply the config during build time or we can have a consistent storage using etcd, for example for /etc

**Comment by smira** [comment]
Are you asking about https://github.com/siderolabs/talos/issues/7791?

**Comment by stevefan1999-personal** [comment]
@smira Absolutely! I have this idea of running off the configurations with a FUSE filesystem like Proxmox, and in that case etcd would be a perfect target (given if etcd won't screw up)
